raster/src/main/scala/geotrellis/raster/render/ColorMap.scala

```diff
@@ -259,6 +263,13 @@ class IntColorMap(breaksToColors: Map[Int, Int], val options: Options = Options.
     h.foreachValue(z => ch.setItem(z, map(z)))
     new IntCachedColorMap(orderedColors, ch, options)
   }
+
+  lazy val breaksString: String = {
```

Comment:
I can't see it being called over and over for one instance of a `ColorMap`. I suppose the GC behaviour for these are:

* `val`: Eagerly evaluate, keep in memory
* `lazy val`: Lazily evaluate, and then keep in memory
* `def`: Lazily evaluate, don't keep in memory

Then perhaps it's best as a `def`.